x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -124,16 +118,8 @@ export const useCasesColumns = ({
               <TruncatedText text={theCase.title} />
             </CaseDetailsLink>
           );
-          return theCase.status !== CaseStatuses.closed ? (
-            caseDetailsLinkComponent
-          ) : (
-            <EuiFlexGroup direction="column" gutterSize="none">
```

Comment:
Was this messing with dark mode? Any idea why we had it in the first place rather than just rendering the `caseDetailsLinkComponent`?